### PR TITLE
Added a fill_gaps method to LightCurve which fills in gaps in time and NaN values with linear intepolation

### PR DESCRIPTION
**Pull request by christinahedges**
It's called `fill_gaps()`, it was actually written by @barentsen. We need to merge this ASAP to get the periodogram stuff from #177 merged.

### CODE REVIEW

**Comment by christinahedges** [comment]
I don't understand the travis/appveyor fails here I think it's to do with me pushing to the wrong remote?

File: lightkurve/lightcurve.py [hunk 1]
```diff
@@ -355,6 +356,46 @@ def remove_nans(self):
         """
         return self[~np.isnan(self.flux)]  # This will return a sliced copy
 
+    def fill_gaps(lc, method='nearest'):
+        """Fill in gaps in time with linear interpolation.
+
+        Parameters
+        ----------
+        method : string {None, ‘backfill’/’bfill’, ‘pad’/’ffill’, ‘nearest’}
```
Comment (johnnyzhang295) [inline lightkurve/lightcurve.py]: Builds are failing because of the single quotes I believe

**Comment by christinahedges** [comment]
@barentsen can you take a look at this and merge it? Johnny is happy! :D

**Comment by barentsen** [comment]
Travis passes.  Merging.  Thanks @christinahedges and @johnnyzhang295!

**Pull request merged**